CMakeLists.txt

```diff
@@ -123,28 +124,31 @@ find_library(HIREDIS_LIBRARY
   PATHS
   third_party/hiredis/install/lib)
 
-find_library(ALIVE_LIBRARY alive2 PATHS "${ALIVE_DIR}/build")
+find_library(ALIVE_LIBRARY alive2 PATHS "${ALIVE_DIR}/build" NO_DEFAULT_PATHS)
 
 find_path(Z3_INCLUDE_DIRECTORY
   NAMES
   z3.h
   z3++.h
   PATHS
-  third_party/z3-install/include)
+  third_party/z3-install/include
+  NO_DEFAULT_PATHS)
 
 include_directories(${Z3_INCLUDE_DIRECTORY})
 
 find_library(Z3_LIBRARY
   NAMES
-  z3
+  libz3.a
   PATHS
-  third_party/z3-install/lib)
+  third_party/z3-install/lib
+  NO_DEFAULT_PATHS)
 
 find_program(Z3
   NAMES
   z3
   PATHS
-  third_party/z3-install/bin)
+  third_party/z3-install/bin
+  NO_DEFAULT_PATHS)
```

Comment:
I looked in the CMake source code and there's no support for something called NO_DEFAULT_PATHS

Reply:
I think it should be NO_DEFAULT_PATH like you had before